tools/gohan_client_bash_completion.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+
+commands="list show create set delete"
+verbosity="0 1 2"
+outputFormat="json table"
+gohanCommandPos=1
+unnamedCommandWords="show set delete"
+getUnnamedProperties()
+{
+schema_id=$1
+OIFS=$IFS
+IFS=$'\n' arr=`${COMP_WORDS[0]} client $schema_id list`
+wasProperties=0
+counter=0
+unnamed=""
+for a in $arr
+do
+        cop=$a
+        replacewhat="-"
+        replacewith=""
+        replacewhat2="+"
+        result="${cop//$replacewhat/$replacewith}"
+        result="${result//$replacewhat2/$replacewith}"
+        let counter=counter+1
+        if [  -z $result ] ; then
+        continue
+        fi
+        if [[ $counter -gt 2 ]];then
+        args=$(echo $a | tr "|" "\n")
+        sub=0
+        for i in $args
+        do
+
+            let sub=sub+1
+            if [ $sub -eq 1 ] && [ ! -z "${i// }" ] ; then
```

Comment:
This works and doesn't depend on the order of columns:
```bash
unnamed="$(${COMP_WORDS[0]} client "$schema_id" list | 
    grep -v -e '+-\++' -e '^[[:space:]]*$' | 
    awk -v col='ID' -F '|' 'NR==1 { for (i=1; i<=NF; i++) if ($i ~ "[[:space:]]*" col "[[:space:]]*") { c=i; break } } NR>1 { print $c }')"
```
(the awk is adapted from https://stackoverflow.com/a/32612999/795068).